lib/email/styles.rb

```diff
@@ -271,11 +271,7 @@ def to_html
       strip_classes_and_ids
       replace_relative_urls
 
-      if SiteSetting.preserve_email_structure_when_styling
-        @fragment.to_html
-      else
-        include_body? ? @fragment.at("body").to_html : @fragment.at("body").children.to_html
```

Comment:
I guess you can delete the `include_body?` method as well. It doesn't seem to be used in core after this change.

Reply:
Good catch.